content/participants/ttjc@tanaypratap/index.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: dummy student
-type: "participant"
-name: "dummy student"
+title: student
+type: "finalist"
+name: "student"
 portfolio: "https://tanaypratap.com"
 github-link: "https://github.com/tanaypratap"
 linkedin-link: "https://linkedin.com/in/tanaypratap"
```

Comment:
Add title as your name, name as your name add your portfolio and social links example:  
title: Kanishk
type: "participantt"
portfolio: "https://gkanishk.github.io"
github-link: "https://github.com/gkanishk"
linkedin-link: "https://linked.com/in/gkanishk"

Do the changes and cross-check ones done.
Also, you need to make a separate file for the finalist.
Refer [faq](https://2020.teamtanay.jobchallenge.dev/finalist-entry/) or #935